lib/internal/Magento/Framework/App/FeedImporterInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedImporterInterface
+{
+
+    /**
+     * @throws \Magento\Framework\Exception\FeedImporterException
+     * @param  array  $data
+     * @param  string $format
+     * @return FeedInterface
+     */
+    public function importArray(array $data, $format = 'atom');
```

Comment:
it's not good from interface point of view, that 'atom' passed as a default parameter.

What other formats does this Importer support? 
It's not clear from the interface. 
And not clear how to check this out. 
Interface should be clear enough to get this knowledge